frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -198,11 +199,12 @@ export const trendsLogic = kea<trendsLogicType<IndexedTrendResult, TrendPeople,
         loadPeople: (
             action: ActionFilter | 'session', // todo, refactor this session string param out
             label: string,
-            date_from?: string | number,
-            date_to?: string | number,
+            date_from: string | number,
+            date_to: string | number,
             breakdown_value?: string,
             saveOriginal?: boolean,
-            searchTerm?: string
+            searchTerm?: string,
+            funnel_step?: number
```

Comment:
It's a bit of a mess in this action, but normally we should stick to `camelCase` in JS, except when really and directly dealing with responses from API endpoint that follow different conventions.

Here the `funnel_step` var starts from the action, so it makes sense to use the JS convention I think.

Reply:
Yeah, I definitely get confused sometimes when to use camel vs snake case in the frontend because it seems like we use snake case in a lot of places on the frontend so we can pass in params like { funnel_step } instead of { funnel_step: funnelStep }